src/ng/directive/ngClass.js

```diff
@@ -162,7 +162,7 @@ function classDirective(name, selector) {
  * @example Example that demonstrates basic bindings via ngClass directive.
    <example>
      <file name="index.html">
-       <p ng-class="{strike: deleted, bold: important, red: error}">Map Syntax Example</p>
+       <p ng-class="{strike: deleted, bold: important, 'red-error': error}">Map Syntax Example</p>
```

Comment:
This is too similar to `red` (and not very intuitive) :)
How about changing it to `'has-error': error`.

Then you need to change the CSS class (to `has-error`) and also update the text on [line 176][1] to `error (apply "has-error" class)`.

[1]: https://github.com/angular/angular.js/pull/12027/files#diff-d0df966e85933ac85b0f511aa14ca126R176